schema/codelists/partyRole.csv

```diff
@@ -1,7 +1,7 @@
 Code,Title,Description,Deprecated
 buyer,Buyer,"The organization aiming to conclude a contract with a supplier or to use the goods, works or services resulting from the contract.",
 procuringEntity,Procuring entity,"The organization managing the contracting process. If an organization is both a buyer and a procuring entity (as can be the case in simple contracting processes), its roles should include 'buyer', but not 'procuringEntity'.",
-supplier,Supplier,"An entity awarded or contracted to provide goods, works or services.",
+supplier,Supplier,"An entity with which a buyer or a procuring entity decided to conclude a contract.",
```

Comment:
It's a bit silly, but the tests require quotation marks to be omitted unless the cell contains a comma.

```suggestion
supplier,Supplier,An entity with which a buyer or a procuring entity decided to conclude a contract.,
```